src/main/java/com/ipiecoles/java/java220/Cadre.java

```diff
@@ -0,0 +1,47 @@
+package com.ipiecoles.java.java220;
+
+import java.util.Objects;
+
+public class Cadre extends Employe{
+
+    private Double coefficient = 1d;
+
+
+    @Override
+    public Double getPrimeAnnuelle() {
+        return Entreprise.primeAnnuelleBase()*coefficient;
+    }
+
+    public Integer getNbConges(){
+        return super.getNbConges() + (int)Math.round(coefficient);
```

Comment:
Comme ça tu arrondis à l'entier supérieur ;)
```suggestion
        return super.getNbConges() + (int)Math.ceil(coefficient);
```